README.md

```diff
@@ -7,6 +7,7 @@
 [![Github tag](https://img.shields.io/github/tag/skyscanner/backpack-android.svg)](https://github.com/Skyscanner/backpack-android)
 [![license](https://img.shields.io/github/license/Skyscanner/backpack-android.svg)](https://github.com/Skyscanner/backpack-android)
 [![platform](https://img.shields.io/badge/platform-android-green.svg)](https://github.com/Skyscanner/backpack-android)
+[![Release](https://jitpack.io/v/skyscanner/backpack-android.svg)](https://jitpack.io/#skyscanner/backpack-android)
```

Comment:
I wonder if this should replace the `tag` badge instead of adding to it. Otherwise we are duplicating the information.

Reply:
makes sense. Changed to tag.